example/sample-koin/src/commonMain/kotlin/composegears/tiamat/sample/koin/viewmodel/SaveableViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package composegears.tiamat.sample.koin.viewmodel
+
+import com.composegears.tiamat.Saveable
+import com.composegears.tiamat.SavedState
+import com.composegears.tiamat.TiamatViewModel
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.isActive
+import kotlinx.coroutines.launch
+
+internal class SaveableViewModel(savedState: SavedState?) : TiamatViewModel(), Saveable {
+    private val _timer = MutableStateFlow(0)
+    val timer = _timer.asStateFlow()
+
+    init {
+        _timer.value = savedState?.get("count") as? Int? ?: 0
```

Comment:
`savedState?.getOrDefault("count",  0)`?

Reply:
![image](https://github.com/user-attachments/assets/fac9ce1d-6aa4-40bb-a4c2-ffbe994f9d01)
Not supported for some targets